### ISSUE TITLE
Mentat logo options

### ISSUE THREAD

**Issue by ccrites** [mozilla/mentat#348]
![m_for_mentat-02](https://cloud.githubusercontent.com/assets/3907094/23288919/387d60de-f9fb-11e6-93b6-5b2edf0f42e0.jpg)

Hello! 
These are 6 possible logos for the Mentat project. I wanted to draw on the idea of "storage" (a bottle, a container, a database, etc.) and some spicy (_some_ pun intended) Dune lore, and settled on the image of a Mentat's vial of spice. Small and efficient, but full of possibility + potential. 

So! If there's a particular version you're drawn to, or a facet of an option that you wish appeared somewhere else, let me know! I'm happy to make changes/mix and match. :)

**Comment by ncalexan** [comment]
Let's label these like `XY` for discussion, like so:

| | |
| --- | --- |
| 11 | 21 |
| 21 | 22 |
| 31 | 32 |

**Comment by ncalexan** [comment]
I personally like 12 and 21.  Something about the symmetry works for me.

**Comment by jsantell** [comment]
Er, some dupes in that matrix:

Let's label these like `XY` for discussion, like so:

| | |
| --- | --- |
| 11 | 21 |
| 12 | 22 |
| 13 | 23 |

**Comment by jsantell** [comment]
I agree with @ncalexan, I like 12 and 21 (the ones with the "drop", presumably a spicy datum) -- some variations could be with the lettermark to the side, or omitted, with the content providing the name "Mentat" with whatever the medium is (see most Webpack or React logo usage); optionally, I wonder what 12 and 21 would look like without the circle badge (maybe as an alternative use).

These look great, thank you @ccrites!

**Comment by ncalexan** [comment]
@jsantell yeah, I typoed that pretty hard.  It's fixed now.

**Comment by ncalexan** [comment]
> ... would look like without the circle badge (maybe as an alternative use).

I personally like the circle badge a lot, because I don't like the "head weighted" look without it.  That is, they look top-heavy without the badge to balance everything out.

**Comment by joewalker** [comment]
I'm also a fan of 21 and 12.

I'm no art nerd, so these opinions should be weighted down heavily - I tried to work out why I preferred those 12/21 - I think the font in 13, 22 and 23 feels like it has style opinions, and often that logo is going to have to fit in with whatever else is going on.

I like the vertical light/shade line down the middle of the bottle. I don't know if you were tempted to give it some contour like this sort of thing:
![Bottle with contour](http://www.eatlogos.com/fashion_logos/png/vector_bottle_logo.png)

But the vertical line made me think of the HTML5 logo, which is kind of a nice hat-tip.

![HTML5 logo](https://www.w3.org/html/logo/downloads/HTML5_Logo_256.png)

Also, I like the orange/blue colors, which may or may not have been deliberate:

![Mozilla logo](https://www.mozilla.org/media/img/firefox/firefox-256.e2c1fc556816.jpg)

I think I'm ending up thinking that 21 is my favorite, followed by 12, followed by 11.

**Comment by rnewman** [comment]
I like the two-color aspects that suggest faceting; I'd be interested in exploring options that grow that, perhaps along the lines of an angled square glass apothecary bottle. Which suggests the Alice in Wonderland connection (apologies for the terrible iPad sketch):

![fullsizerender jpg](https://cloud.githubusercontent.com/assets/91722/23319433/5f207bfa-fa8b-11e6-85cb-2b5b9fa34668.jpeg)

I do like the orange and blue combined (my two favorite colors!), but the nerd in me is compelled to note that mentats drink Sapho juice, which is red. No need for 'historical' accuracy in this case, though!

**Comment by jsantell** [comment]
Looks like ac724bb accidentally closed this

**Comment by ncalexan** [comment]
> Looks like ac724bb accidentally closed this

Yep, and I force pushed to fix the message.  Sorry all!

**Comment by ccrites** [comment]
Hi all! 
Thank you so much for the feedback! 
I can address specific comments later on; at the moment I'm packing/prepping to vacate my apartment while it's renovated during March. I'll have more time to respond + work on iterations starting tomorrow, when I officially (and ideally temporarily) become a **roaming designer**.

Anyway! Saw there were a few comments re: the bottle looking a bit top heavy/not being balanced without the circle background. I did a few shape variations - let me know what you think! 
![Uploading mentatbottleoptions-03.png…]()

**Comment by rnewman** [comment]
Gah, I'm so sorry for dropping the ball on this, @ccrites! I thought I'd replied to this, but I hadn't.

We're interested in moving forward with #2, if you still have the time to work this up to something you'd like to see gracing laptops and lampposts in sticker form. (If your circumstances have changed in the last month, that's totally understandable, of course.)

And thanks again!

**Comment by ccrites** [comment]
@rnewman Hey Richard! Thanks for the update! Circumstances have changed a bit for me (aka I got a full time job, YAY) but I should have some time this weekend to finalize #2 for review. I'll keep ya posted! I *would* really like to see this as a sticker tbh.

**Comment by ncalexan** [comment]
> @rnewman Hey Richard! Thanks for the update! Circumstances have changed a bit for me (aka I got a full time job, YAY)

Awesome! 🎆 🍾 

> but I should have some time this weekend to finalize #2 for review. I'll keep ya posted! I would really like to see this as a sticker tbh.

Also good :)

**Comment by ccrites** [comment]
![menat_orange-01](https://cloud.githubusercontent.com/assets/3907094/25031094/323d71fa-207f-11e7-94a5-1c182c0b5455.png)

This is bottle 2 from above with some slight updates/gradients.
